app/services/service_provider_seeder.rb

```diff
@@ -25,11 +26,20 @@ def run
 
   attr_reader :rails_env, :deploy_env
 
+  # rubocop:disable Metrics/AbcSize
+  #:reek:DuplicateMethodCall :reek:TooManyStatements
   def service_providers
     file = remote_setting || Rails.root.join('config', 'service_providers.yml').read
     content = ERB.new(file).result
-    YAML.safe_load(content).fetch(rails_env, {})
+    YAML.safe_load(content, aliases: true).fetch(rails_env)
```

Comment:
I would advise against using aliases in service_providers.yml. YAML aliases can promote unexpected behavior and can be difficult to follow. Almost any use case for aliases can be accomplished by creating a mechanism for more explicit defaults or inheritance.

e.g. In this case, you could replace a nil value or `:default` value with the default text.